compiler-rt/lib/scudo/standalone/wrappers_c.inc

```diff
@@ -252,13 +252,10 @@ INTERFACE WEAK int SCUDO_PREFIX(mallopt)(int param, int value) {
       // introduced by interval transition.
       SCUDO_ALLOCATOR.releaseToOS(scudo::ReleaseToOS::Force);
 
-      if (value == 0) {
-        // Will set the release values to their minimum values.
-        value = INT32_MIN;
-      } else {
-        // Will set the release values to their maximum values.
+      // The values allowed on Android are {-1, 0, 1}. "1" means the longest
+      // interval.
+      if (value == 1)
```

Comment:
Should this check be value > 0? That more closely matches what it did before.

Reply:
I'm thinking instead of ">=", maybe we can assert the range of values? given that only three values are valid.  (And the description also says 0 and 1 are only allowed numbers)

Reply:
That would be fine with me. We sanitize the values when coming through Android anyway, so verifying a -1, 0, or 1 is probably a better way to do this.

Reply:
Done. CHECK added.